routes/courses.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+// @route   GET /courses
+// @desc    Get a messgae
```

Comment:
If you are going to leave a comment, make sure they are up to date. Comments are one of the easiest things to not update and becomes mis-informative.